consensus/src/liveness/round_state.rs

```diff
@@ -157,8 +159,10 @@ pub struct RoundState {
     time_service: Arc<dyn TimeService>,
     // To send local timeout events to the subscriber (e.g., SMR)
     timeout_sender: aptos_channels::Sender<Round>,
-    // Votes received fot the current round.
+    // Votes received for the current round.
     pending_votes: PendingVotes,
+    // OrderVotes received for the last 2 rounds.
+    pending_order_votes: PendingOrderVotes,
```

Comment:
it feels awkward to have it in round state for multiple rounds, better for round manager